app/src/main/aidl/io/left/meshim/services/IMeshIMService.aidl

```diff
@@ -19,7 +19,7 @@ interface IMeshIMService {
 
     void registerActivityCallback(in IActivity callback);
 
-    void sendTextMessage(in User recipient, in String message);
+    void sendTextMessage(in User recipient, in String message, in String filePath, in String fileName);
```

Comment:
Maybe a new "sendFile" method, or just a second method with more arguments would be better than requiring these every time this is called?